### PR TITLE
using hub

### PR DESCRIPTION
**Pull request by sgrg1**
using hub to call api .

### CODE REVIEW

**Comment by martypowell** [comment]
> using hub to call api .

have you tested this?

**Comment by sgrg1** [comment]
It does not work from local machine.

**Comment by martypowell** [comment]
> It does not work from local machine.

I'm not sure what this means local machine. You are saying it doesn't work on your work pc right?

**Comment by sgrg1** [comment]
yes, my pc at work. I was just wondering , since structure content does not have hub , how does it work?

**Comment by martypowell** [comment]
@sgrg1 the news types need to be added to the hub, please follow the readme, will send over chat.

File: src/App.js [hunk 1]
```diff
@@ -9,16 +9,16 @@ import React from "react";
...
-  "https://services.baltimorecountymd.gov/hub/api/hub/news/police/police";
+  "https://services.baltimorecountymd.gov/hub/api/hub/structuredContent/police";
 
 // HACK - the Config utiltiy does not account for beta.
 // TODO: This will need to be addressed when we get closer to launch
 const localApiRoot =
   window.location.hostname.indexOf("beta") > -1
     ? testApiRoot
-    : "https://localhost:44393/api/hub/news/police/police";
+    : "https://localhost:44393/api/hub/structuredContent/police";
```
Comment (martypowell) [inline src/App.js:21]: `/structureContent/news/police` i believe. I think i may have told you the wrong url, thats why i wouldn't trust me. Would also recommend testing prior to submitting the pull request.

Comment (sgrg1) [reply src/App.js]: done. test works

**Pull request merged**